lib/custom_a11y_rules.js

```diff
@@ -156,6 +171,26 @@ var customRules = {
             evaluate: function(node, options) {
                 return axe.utils.isHidden(node);
             },
-        }
+        },
+        {
+            'id': 'section-heading-first-child',
+            'metadata': {
+                'impact': 'minor',
```

Comment:
Correct, these - although minor - would fail which would in turn cause Jenkins to fail. I know there are a bunch of warnings, but Christine wasn't taking those into account for the tests or Jenkins. There may be a way to equate "minor" with "warning", but then what's the point since they wouldn't be seen by developers anyway? Open to feedback or thoughts here, but my guess is it's either "on" and triggering relevant Jenkins failures or "off" and not.

The "impact" (minor, major, critical) is more to give developers a sense of severity, but a fail is still a fail.

Again though, I'd have to do some research in this area.

The other thought would be to include these tests, but turn them off (similar to how Christine did it with the video checks). So we'd have them, but they wouldn't be run.